### PR TITLE
adding info from google doc

### PR DESCRIPTION
**Pull request by jukent**
First pass at updating content on site

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,15 +6,30 @@
 
 ## What is it?
 
-_One paragraph description of the event_
+Project Pythia invites you to join a Cookbook Cook-off.  Pythia Cookbooks (https://cookbooks.projectpythia.org) are crowd-sourced collections of advanced, domain-specific tutorials and exemplar workflows, building upon our Pythia Foundations (https://foundations.projectpythia.org) tutorials. Cookbooks are supported by a rich GitHub-based infrastructure enabling collaborative authoring and automated health-checking to ensure reproducibility.
```
Comment (brian-rose) [inline README.md]:
```suggestion
Project Pythia invites you to join a Cookbook Cook-off.  [Pythia Cookbooks](https://cookbooks.projectpythia.org) are crowd-sourced collections of advanced, domain-specific tutorials and exemplar workflows, building upon our [Pythia Foundations](https://foundations.projectpythia.org) tutorials. Cookbooks are supported by a rich GitHub-based infrastructure enabling collaborative authoring and automated health-checking to ensure reproducibility.
```
Suggest using clickable links here

File: faq.md [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
+Unfortunately, we were unable to secure travel funds for this event.
 
 ## Can I participate remotely?
 
 Yes! Our meeting rooms will be equipped for hybrid events. We will have at least two synchronous plenary sessions each day so remote participants will be able to present work to the whole group.
 
 ## Is there a reserved hotel block for this event?
 
-Yes! _Need to update this_
+Yes! We requested 20 rooms at the _______.
```
Comment (brian-rose) [inline faq.md]: Is the hotel still unknown?

**Review by brian-rose — COMMENTED** [review]
Looks good @jukent, I just left a couple of comments / suggestions.

File: faq.md (hunk 2 above)
Comment (jukent) [reply faq.md]: Yes. I asked Taysia for an update on it yesterday, so hopefully we hear back soon.

**Review by brian-rose — APPROVED** [review]

**Pull request merged**